### DISCUSSION TITLE
2024-06-19(수)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#181] [데일리스크럼]
아래 내용을 복사해 작성해주세요.
```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- 온보딩 페이지 / put API 요청 / 그래픽 삽입
- 프로필 수정 페이지 put API 요청

### 해야할 일
- signup 페이지 닉네임 중복 검사 구현
- 프로필 이미지 삽입 기능 구현 / #173 

### 어려운 일
- 카카오 로그인 OAuth 이슈(새로고침 해야 데이터 패치) 자꾸 발생하네요...

**Comment by minyoung0503** [comment]
### 완료한 일
myReviewWrite API 연결
option BottomSheet API 연결(option 관련된 API만, 수량 기능 x)

### 해야할 일
myReviewDetail API GET
option BottomSheet UI

### 어려운 일
option BottomSheet
myReviewWrite API 에러 처리

**Comment by devwqc** [comment]
### 완료한 일
- 라우팅 시 `Loading` 띄우기

### 해야할 일
- 홈, 검색, 찜 페이지 작업

### 어려운 일
- 파이팅!!💪

**Comment by q45402sk** [comment]
### 완료한 일
myDeliveryPage 
상품상세 (pr 올릴 예정)

### 해야할 일
purchaseDeliveryPage

### 어려운 일
